Multisig/UI/Settings/Passcode/EnterPasscodeViewController.swift

```diff
@@ -69,6 +69,11 @@ class EnterPasscodeViewController: PasscodeViewController {
 
         if isCorrect {
             passcodeCompletion(true, false, text)
+            do {
+                try onPasscodeEnter(text)
+            } catch {
+                LogService.shared.error(error.localizedDescription)
```

Comment:
here I think we need to call onError callback instead

Reply:
See latest commit.